functions.php

```diff
@@ -100,6 +100,18 @@ function getUser($nick, $conn = null, $client = null) {
   return $user;
 }
 
+function deleteUser($nick, $conn = null, $client = null) {
+  if ($conn) {
+    $sql = "delete from users where login = '$nick'; ";
```

Comment:
I'm not intimately familiar with PHP but I'm pretty sure this is vulnerable to SQL injection as it's blindly concatenating the user supplied nick. It should be escaped or (*much better*) be supplied as a bind variable (i.e. parameterized statement).